agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -1664,5 +1664,20 @@ bool main_thread::init_dhcp_monitor()
     return true;
 }
 
+std::string main_thread::get_sta_iface(int slave_num)
```

Comment:
If you are not accessing other class parameters here, better make this static instead of class private.

Reply:
but I am accessing the `config.hostap_iface[]` which is a class parameter.

I changed the API to get hostap_iface and the API is now static.

I changed the corresponding use in the following commit titled "platform_manager - add sta_ifaces to iface_status_map in init"